plugins/github/src/org/jetbrains/plugins/github/pullrequest/GHPRComponentFactory.kt

```diff
@@ -242,6 +242,8 @@ internal class GHPRComponentFactory(private val project: Project) {
       installSelectionSaver(it, listSelectionHolder)
       val shortcuts = CompositeShortcutSet(CommonShortcuts.ENTER, CommonShortcuts.DOUBLE_CLICK_1)
       EmptyAction.registerWithShortcutSet("Github.PullRequest.Timeline.Show", shortcuts, it)
+      val search = ListSpeedSearch(it) { item -> item.title }
+      search.comparator = SpeedSearchComparator(false)
```

Comment:
You don't need to set this comparator, since it is already the default.

Reply:
Thanks